lib/foreman_plugin_template/engine.rb

```diff
@@ -18,9 +18,13 @@ class Engine < ::Rails::Engine
       Foreman::Plugin.register :foreman_plugin_template do
         requires_foreman '>= 1.16'
 
+        # Add Global JS file for extending foreman-core components and routes
+        register_global_js_file 'fills'
```

Comment:
the file is named `fills_index.js`, so i think this needs to change

Reply:
the `_index` suffix is added automatically.